jsonschema-core.xml

```diff
@@ -2095,6 +2101,442 @@
+            <t>
+                JSON Schema is defined to be platform-independent.  As such, to increase compatibility
+                across platforms, implementations SHOULD conform to a standard validation output
+                format.  This section describes the minimum requirements that consumers will need to
+                properly interpret validation results.
+            </t>
+
+            <section title="Format">
+                <t>
+                    JSON Schema output is defined using JSON.  Implementations MAY deviate from this
+                    as supported by their specific languages and platforms, however it is RECOMMENDED
+                    that the output be convertible to the JSON format defined herein via serialization
+                    or other means.
+                </t>
+            </section>
+
+            <section title="Output Levels">
+                <t>
+                    This specification defines four levels of output.  See the *Output Structure*
+                    section for the requirements of each level.
+                    <list>
+                        <t>
+                            Flag - A boolean which simply indicates whether the schema was
+                            validated.
+                        </t>
+                        <t>
+                            Basic - Provides validation information in a flat list structure.
+                        </t>
+                        <t>
+                            Detailed - Provides validation information in a condensed hierarchical
+                            structure based on the structure of the schema.
+                        </t>
+                        <t>
+                            Verbose - Provides validation information in an uncondensed hierarchical
+                            structure that matches the exact structure of the schema.
+                        </t>
+                    </list>
+                    Implementations SHOULD specify which levels of validation they support.
+                </t>
+
+            </section>
+
+            <section title="Minimum Information">
+                <t>
+                    Beyond the simplistic "Flag" output, additional information is useful to aid in
+                    debugging a schema or instance.  Each sub-result SHOULD contain the information
+                    contained within this section at a minimum.
+                </t>
+                <t>
+                    A single object that contains all of these components is considered an
+                    output unit.
+                </t>
+                <t>
+                    Implementations MAY elect to provide additional information.
+                </t>
+
+                <section title="Keyword Relative Location">
+                    <t>
+                        The relative location of the validating keyword that follows the validation
```

Comment:
Can we define the term "validation path" in section 4.3.4 Lexical Scope and Dynamic Scope?  The validation path essentially captures the dynamic scope, and it would be good to make that clear.

Reply:
✔️